### DISCUSSION TITLE
Using next/image (WebP) in production

### DISCUSSION THREAD

**Discussion by doptster** [vercel/next.js#18656] [Help] [answered]
Hi, I want to ask is it safe to use the new converted image format (WebP) by next/image in production? 
Does it have any fallback in case the browser is not supported?

**Comment by AmazingDevTeam** [comment]
I have encountered some issue where safari on iphone 6 is not able to view webp images.

**Comment by timneutkens** [comment, marked as answer]
webp is only served to browsers that support it (browsers publish what formats they support). All features in Next.js are used production by many users.

**Reply by doptster** [reply]
thanks, I will retest on those old browsers to verify.

**Comment by dohomi** [comment]
@timneutkens I think it would be helpful if the section of https://nextjs.org/docs/api-reference/next/image#loader would mention a technique to be used how to determine webP support. Ultimately it would be great if the loader options would have `hasWebP` boolean to be read. There are image proxies out there which does not automatically determine the support based on the request and need a directive to serve webP format.
